<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialTest</class>
 <widget class="QWidget" name="SerialTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Command Tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTop">
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Serial Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="COMPort"/>
     </item>
     <item>
      <widget class="QComboBox" name="Baudrate"/>
     </item>
     <item>
      <widget class="QComboBox" name="Parity"/>
     </item>
     <item>
      <widget class="QComboBox" name="StopBit"/>
     </item>
     <item>
      <widget class="QComboBox" name="NData"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoadXml">
       <property name="text">
        <string>Load XML</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetCommands">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Test</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Command</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Expected</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value (Hex)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="test">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Single">
       <property name="text">
        <string>Single</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Loop">
       <property name="text">
        <string>Loop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutLog">
     <item>
      <widget class="QTextEdit" name="textEditLog">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Log will appear here...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutRight">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="Timestamp">
           <property name="text">
            <string>Timestamp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Connect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_clear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Log">
           <property name="text">
            <string>Log</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
